internal/render/hpa.go

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of K9s
+
+package render
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/derailed/tcell/v2"
+)
+
+// HorizontalPodAutoscaler renders a K8s HorizontalPodAutoscaler to screen.
+type HorizontalPodAutoscaler struct {
+	Generic
+}
+
+// ColorerFunc colors a resource row.
+func (hpa HorizontalPodAutoscaler) ColorerFunc() ColorerFunc {
+	return func(ns string, h Header, re RowEvent) tcell.Color {
+		c := DefaultColorer(ns, h, re)
+
+		maxPodsS := strings.TrimSpace(re.Row.Fields[h.IndexOf("MAXPODS", true)])
```

Comment:
We should validate `IndexOf` returns a valid index ie within the Fields range.

Reply:
hey @derailed, thanks for reviewing my PR. I'll make the changes!

Reply:
hey @derailed , could you take another look? Can I mark it as resolved?